### DISCUSSION TITLE
Generated PDFs don't look exactly like the Word documents given to the API. Messing up PDF parser result

### DISCUSSION THREAD

**Discussion by andre2xu** [gotenberg/gotenberg#906] [Q&A] [answered]
Hi, I'm using LibreOffice in the [PHP client for Gotenberg](https://github.com/gotenberg/gotenberg-php) to convert Microsoft Word files to PDF, and this [parser](https://github.com/smalot/pdfparser) to get their text. However, the parser is giving me some weird strings that don't look like contents of the PDFs at all. I figured it was because the PDFs generated by LibreOffice look different compared to their Word document counterpart (see below). How do I configure LibreOffice properly so that the resulting PDFs match the Word documents? (I want a behaviour similar to the 'save as' feature in MS Word)

FYI: The Word documents I'm using are the free resume templates provided by Microsoft ([https://create.microsoft.com/en-us/templates/resumes](https://create.microsoft.com/en-us/templates/resumes)). Their extensions are `.docx` and their content type is `application/vnd.openxmlformats-officedocument.wordprocessingml.document`. I don't know if this information is useful but I'm giving it just in case.

Example 1
![libreoffice result 1](https://github.com/gotenberg/gotenberg/assets/71770525/c99fdd6b-fff1-4214-9e38-5127049bc3fa)

Example 2
![libreoffice result 2](https://github.com/gotenberg/gotenberg/assets/71770525/0c7cd2f7-9cbe-4da1-ac6a-2bba7d4db87a)

My LibreOffice code:

```php
$pdf_conversion_request = Gotenberg::libreOffice('http://gotenberg:3000')->convert(Stream::path($doc_file_path));

try {
    $pdf_name = Gotenberg::save($pdf_conversion_request, $folder_path);
}
catch (GotenbergApiErroed $error) {
    var_dump($error);
}
```

**Comment by andre2xu** [comment, marked as answer]
Never mind I found the solution. All I had to do was disable the exporting of form fields. I don't know why that is enabled by default.
```php
$pdf_conversion_request = Gotenberg::libreOffice('http://gotenberg:3000')->exportFormFields(false)->convert(Stream::path($doc_file_path));
```

**Reply by gulien** [reply]
@andre2xu yeah I’m using the same defaults as LibreOffice, but maybe in the next major version of Gotenberg I’d use defaults that make more sense for the end user 🤔 thanks for the feedback!

**Reply by omni-htg** [reply]
Hey @gulien ! Mind if I piggyback of this thread to request `IsSkipEmptyPages` also falls into that category? In the context of "converting", IMO it makes more sense to set it to `true` instead of it's `false` default from LibreOffice 😃
